README.md

```diff
@@ -55,35 +55,27 @@ To run tests on your local machine, first you need to start the frontend server:
 yarn dev
 ```
 
-Then you need to run the tests using
+If you plan to run tests with `CYPRESS_AGORIC_NET=local`, you must start the `a3p` chain beforehand. To do this, use the following command:
```

Comment:
Seems like `yarn test:e2e` is missing now in the new instructions. 
Also missing where network can be emerynet, xnet etc?